models/summary.go

```diff
@@ -77,6 +77,12 @@ type SummaryParams struct {
 	Recompute bool
 }
 
+type DailyProjectStat struct {
```

Comment:
Minor one: we consider a "model" something that's persisted on the database or at least part of Wakapi's core logic. The `DailyProjectStat`, however, is only another "view" to the data (in a format that's later required for rendering the chart), so I'd rather call this a "view model", analogously to, for example `ProjectsViewModel`, `LeaderboardViewModel`, etc.